app/controllers/sections/consignee/ConsigneeIndexController.scala

```diff
@@ -54,7 +54,7 @@ class ConsigneeIndexController @Inject()(override val messagesApi: MessagesApi,
               if (shouldStartFlowFromConsigneeExemptOrganisation(destinationTypePageAnswer)) {
                 Redirect(controllers.sections.consignee.routes.ConsigneeExemptOrganisationController.onPageLoad(ern, draftId, NormalMode))
               } else if (shouldStartFlowFromConsigneeExportUkEu(ur.userTypeFromErn, destinationTypePageAnswer)) {
```

Comment:
It's worth changing this method name to prevent confusion (as the page it refers to has been deleted)